.vscode/extensions.json

```diff
@@ -3,7 +3,8 @@
 	// for the documentation about the extensions.json format
 	"recommendations": [
 		"dbaeumer.vscode-eslint",
-		"EditorConfig.EditorConfig",
```

Comment:
We still have an .editorconfig in the repo. Wouldn't it be useful to have this?

Reply:
Valid point. Retained it.